### PR TITLE
build and publish latest images

### PR DESCRIPTION
**Pull request by InoMurko**

### CODE REVIEW

**Review by CAPtheorem — APPROVED** [review]
Looks great - suggest renaming so it's clear that its part of the omgx stack

**Comment by InoMurko** [comment]
> Looks great - suggest renaming so it's clear that its part of the omgx stack

done!

File: .github/workflows/omgx-publish-develop.yml [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./ops/docker/Dockerfile.deployer
+          push: true
+          tags: omgx/deployer:develop
+
+  integration_tests:
```
Comment (CAPtheorem) [inline .github/workflows/omgx-publish-develop.yml:213]: suggest to call this integration-tests for consistency with naming convention - seems minor but broke the stack before

**Review by souradeep-das — APPROVED** [review]

File: .github/workflows/omgx-publish-develop.yml (hunk 1 above)
Comment (InoMurko) [reply .github/workflows/omgx-publish-develop.yml:213]: consistency with what?
https://github.com/omgnetwork/optimism/blob/develop/.github/workflows/publish-canary.yml#L238
https://github.com/omgnetwork/optimism/blob/develop/.github/workflows/release.yml#L238

Comment (InoMurko) [reply .github/workflows/omgx-publish-develop.yml:213]: the links point to existing `integration_tests`

**Pull request merged**